Source/WebCore/PAL/pal/PALSwift/CryptoKitShim.swift

```diff
@@ -188,108 +193,42 @@ public final class AesKw: NSObject {
     }
 }  // AesKw
 
-@objc(WebCryptoDigestSize)
-public enum DigestSize: Int {
-    case sha1 = 20
-    case sha256 = 32
-    case sha384 = 48
-    case sha512 = 64
+extension HashFunction {
+    mutating func update(data: SpanUInt8) {
+        self.update(
+            bufferPointer: UnsafeRawBufferPointer(
+                start: data.__dataUnsafe(), count: data.size()))
+    }
```

Comment:
To demonstrate our discipline of programming using safe constructs, and our expectation that future APIs will expose safe constructs, I think it's good for overlays that shim unsafe APIs like this to live in a separate file. Maybe "CryptoKitOverlay.swift", or we can put them all together in "UnsafeOverlays.swift".

If you'd like, you can make this change in the follow-up patch that removes the other unsafe pointers from this file.